src/types.ts

```diff
@@ -68,14 +68,14 @@ export interface ActionPluginPayload {
   required_parameters: { [key: string]: any }
 }
 
-export interface User {
-  first_name?: string
-  last_name?: string
-  username: string
-  date_of_birth?: Date
-  password_hash: string
-  email: string
-}
+// export interface User {
+//   first_name?: string
+//   last_name?: string
+//   username: string
+//   date_of_birth?: Date
+//   password_hash: string
+//   email: string
+// }
```

Comment:
Better delete than leave comments. If it is required the code can be checked in a previous version.